tests/test_jinja_macros.py

```diff
@@ -357,6 +368,37 @@ def test_question_above_answer(
         assert "Test Question" in rendered_html, "Question not found"
         assert expected in rendered_html, "Answer not found"
 
+    def test_question_above_answer_html(self, request_ctx):
+        meta = AboveQuestionAnswerPairHtml.from_dict(
+            {
+                "question": "Test Caption",
+                "answer": "<p>This is</p> <strong>free text answer</strong>",
+            }
+        )
+
+        rendered_html = render_template_string(
+            "{{ question_above_answer_html(meta) }}",
+            question_above_answer_html=get_template_attribute(
+                "macros/theme/question_above_answer_html.jinja2",
+                "question_above_answer_html",
+            ),
+            meta=meta,
+        )
+
+        soup = BeautifulSoup(rendered_html, "html.parser")
+
+        data_url = "data:text/html," + quote(
+            "<html><body>" + rendered_html + "</body></html>"
+        )
+        webbrowser.get("chrome").open_new_tab(data_url)
```

Comment:
when are we using the webbrowser part here?

Reply:
Just to ensure the formatting of the HTML tags @Adam-W1

Reply:
okay i think this works for our local set-up but might cause issues elsewhere and we dont run an assertion on this - I think beautifulSoup probs covers it?

Reply:
Good point! I will remove it now. Yes beautiful soup covers it all @Adam-W1